api/buffer/design.go

```diff
@@ -492,6 +492,8 @@ var Mapping = Type("Mapping", func() {
 	})
 	Attribute("columns", ArrayOf(Column), func() {
 		Description("List of export column mappings. An export may have a maximum of 50 columns.")
+		MinLength(1)
+		MaxLength(50)
 		Example([]any{ExampleColumnTypeBody()})
```

Comment:
Tuto validaciu mame sice v `model`, ale ten sa uklada az uplne nakoniec, a ono to spadne skor, pri vytvarani tabulky.

Reply:
Tj. ked v request-e neboli `columns` vobec, tak to zvalidovalo spravne, ale ked tam Vojta dal prazdne array, tak to cez validaciu v goa preslo dalej.